gravitee-apim-console-webui/src/entities/application-general/application-general.fixture.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ApplicationDetails } from '../../management/application/details/general/general-ng/applicationDetails';
+
+export function fakeGeneralList(attributes?: Partial<ApplicationDetails>): ApplicationDetails {
+  const base: ApplicationDetails = {
+    id: 'id_test',
+    name: 'test name',
+    description: 'test description',
+    domain: 'test domain',
+    status: 'test status',
+    type: 'SIMPLE',
+    origin: 'test origin',
+    created_at: 120000,
+    updated_at: 140000,
+    disable_membership_notifications: false,
+    api_key_mode: 'test mode',
+    value: 'test value',
+    owner: {
+      id: 'owner id',
+      email: 'owner email',
+      displayName: 'owner name',
+      type: 'owner type',
+    },
+    settings: {
+      app: {},
+    },
+    picture: 'link picture',
+    background: 'link background',
+  };
+
+  return {
+    ...base,
+    ...attributes,
+  };
+}
+
+export function fakeUpdateGeneraList(attributes?: Partial<ApplicationDetails>): ApplicationDetails {
```

Comment:
why create a new one. WDYT to use fakeApplicationDetails instead ?

Reply:
Done